### ISSUE TITLE
Accessibility - Show Data is not announced to screen reader

### ISSUE THREAD

**Issue by chmnata** [CityofToronto/bdit_cot-vfh#56]
Issue

When the 'Show Data' button is pressed, no announcement is sent to screen readers the raw data is displayed on the page.  Screen reader users need to know the content has been updated.

Recommendation

Use a WAI-ARIA live region to inform screen reader users what item is currently shown.

Locate the WAI-ARIA live region at the top of the page which is part of the COREJS framework.  

```<div class="js-aria-live sr-only" role="status" aria-live="polite"></div>```

Inject the announcement to the WAI-ARIA live region after the 'Show Data' is pressed.  "Showing Monday - trips per hour and Drivers per hour" 

```<div class="js-aria-live sr-only" role="status" aria-live="polite">Showing Monday - trips per hour and Drivers per hour</div>```
